app/(auth)/sign-up/page.tsx

```diff
@@ -0,0 +1,126 @@
+        {icon}
+        {/* <FcGoogle size={18} />{" "} */}
+        <span className="text-sm font-medium">{name}</span>
+      </div>
+    );
+  };
+
+  return (
+    <FormPageTemplate redirectTo="sign-in">
+      <Formik
+        onSubmit={onSubmitHandler}
+        initialValues={signUpFormDetails}
+        validationSchema={signUpValidationSchema}
+      >
+        {({ handleSubmit, errors, touched }) => (
+          <Form>
+            <div className="flex flex-col gap-6">
+              <div>
+                <InputField
+                  name={email.name}
+                  label={email.label}
+                  extraClasses="w-[70vw] sm:w-[480px] md:w-[550px] lg:w-[632px]"
+                  placeholder={email.placeholder}
+                  required={true}
+                  error={errors?.email}
+                  touched={touched?.email}
+                />
+              </div>
+
+              <div>
+                <InputField
+                  name={password.name}
+                  label={password.label}
+                  extraClasses="w-[70vw] sm:w-[480px] md:w-[550px] lg:w-[632px]"
+                  placeholder={password.placeholder}
+                  required={true}
+                  error={errors?.password}
+                  touched={touched?.password}
+                  isPassword={true}
+                />
+              </div>
+
+              <div>
+                <InputField
+                  name={confirmPassword.name}
+                  label={confirmPassword.label}
+                  extraClasses="w-[70vw] sm:w-[480px] md:w-[550px] lg:w-[632px]"
+                  placeholder={confirmPassword.placeholder}
+                  required={true}
+                  error={errors?.confirmPassword}
+                  touched={touched?.confirmPassword}
+                  isPassword={true}
+                />
+              </div>
+            </div>
+
+            <div className="pb-[30px] pt-[34px]">
+              <Button
+                // disabled={isLoading}
+                className="w-[100%] sm:w-[15rem]"
+                variant="default"
+                onClick={() => handleSubmit()}
+              >
+                Sign Up
+              </Button>
+            </div>
+
+            <div className="flex flex-col gap-6">
+              <div className="mb-4 flex items-center">
+                <span
+                  className={`text-sm font-normal text-themeLightGray ${inter.className}`}
+                >
+                  Or signup with
+                </span>
+                <div className="ml-1 h-[1px] flex-grow bg-themeLightGray"></div>
+              </div>
+
+              <div className="flex flex-wrap gap-[10px]">
+                {socialLink(<FcGoogle size={18} />, "Google")}
+
+                {socialLink(<SiSlack size={18} />, "Slack")}
+
+                {socialLink(
+                  <FaJira size={18} className="text-[#2584FF]" />,
```

Comment:
define color in tailwind config.

Reply:
It's just used one time over here

Reply:
@imrankhan-stixor there shouldn't be any unique color. If this is a 1-off, bring this to Waseeq's attention so that he can fix it in the design. Every color in the website should be properly defined on the FIGMA and be initialized in the tailwind config.

Please not that we will also add a feature later on down the line where people can customize theme colors. This will be an issue at that point too.